Dockerfile

```diff
@@ -46,7 +46,7 @@ COPY unstructured_inference unstructured_inference
 RUN python3.8 -m pip install pip==${PIP_VERSION} \
   && pip3.8 install  --no-cache  -r requirements-base.txt \
   && pip3.8 install  --no-cache  -r requirements-dev.txt \
-  && pip3.8 install --no-cache "detectron2@git+https://github.com/facebookresearch/detectron2.git@v0.6#egg=detectron2" \
+  && pip3.8 install --no-cache "detectron2@git+https://github.com/facebookresearch/detectron2.git@e2ce8dc1ab097891395d324abaffe9cf298503d1#egg=detectron2" \
```

Comment:
actually, since we're slinging this pip install all over the place, can we truncate the hash, e.g.:

```
pip3.8 install --no-cache "detectron2@git+https://github.com/facebookresearch/detectron2.git@e2ce8dc#egg=detectron2"
```

Reply:
Done in [17534b2](https://github.com/Unstructured-IO/unstructured-inference/pull/79/commits/17534b299359535c1164adce7bf8e1a1027b1101).